docs/news.rst

```diff
@@ -3,6 +3,158 @@
 Release notes
 =============
 
+1.1.0
+-----
+
+This 1.1 release brings a lot of interesting features and bug fixes:
+
+- Scrapy 1.1 has basic Python 3 support (requires Twisted >= 15.5). See
```

Comment:
I prefer "beta Python 3 support" than "basic" because it works but it is not considered production ready.